src/include/platform/PersistedStorage.h

```diff
@@ -87,6 +91,139 @@ CHIP_ERROR Read(Key aKey, uint32_t & aValue);
  */
 CHIP_ERROR Write(Key aKey, uint32_t aValue);
 
+class KeyValueStore
+{
+public:
+    virtual ~KeyValueStore() = default;
```

Comment:
Why virtual?

Reply:
The actual implementation will extend this class and provide the platform specific implementation.